deepchecks/utils/performance/error_model.py

```diff
@@ -179,7 +179,9 @@ def error_model_display(error_fi: pd.Series,
 
             # Partition data into two groups - weak and ok:
 
-            in_segment_indicis = cum_sum_ratio <= min_segment_size
+            # The weak segment is all the weakest categories accumulated together until they comprise at least
+            # min_segment_size of the total data.
```

Comment:
I think the phrasing here can use some touch-ups + be clearer, but for that I still need to understand what's happening there 😅